apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -1104,6 +1104,73 @@ def create_extension(form, iteration, sub_data):
         if conn is not None:
             conn.close()
 
+def update_extension(form):
+    cur = None
+    conn = None
+    try:
+        conn = psycopg2.connect(
```

Comment:
Optional: This is not specific to this PR, may be we can track in a future task?  Use util method to connect to DB, instead of every method building a connection. This leads to smaller code base and makes it easier to read code that is not boilerplate stuff.

Reply:
Great idea!

Reply:
Created a follow on task for the Portals/APCD group to refactor this. Thank you! https://jira.tacc.utexas.edu/browse/WP-180